alpha/apps/kaltura/modules/extwidget/actions/embedPlaykitJsAction.class.php

```diff
@@ -68,6 +67,11 @@ public function execute()
 
 		$lastModified = $this->getLastModified($bundleContent);
 
+		if($this->getRequestParameter(v2Tov7Utils::V2TOV7_PARAM_NAME))
+		{
+			$bundleContent .= PHP_EOL . v2Tov7Utils::getBundledFacade();
```

Comment:
You ar not checking the exception that might be thrown from getBundledFacade is this intended?